README.md

```diff
@@ -2,25 +2,27 @@
 
 ## What is NGitLab?
 
-*NGitLab* is a .NET REST client implementation of GitLab API with no external dependencies.
+`NGitLab` is a .NET REST client implementation for the GitLab API.
 
 ## Usage
 
-It's a wrapper of REST api. Read the [GitLab docs](https://github.com/gitlabhq/gitlabhq/tree/master/doc/api) and start using by creating a GitLabClient instance:
+Start by creating a `GitLabClient` instance:
 
 ```csharp
 var client = new GitLabClient("https://mygitlab.example.com", "your_private_token");
 ```
 
 Then use its properties. You can obtain the private token in your account page. You may want to create a custom user for the API usage.
 
+For further info about the GitLab API, refer to [the official documentation](https://docs.gitlab.com/ee/api/rest/)
+
 ## Where can I get it?
 
 Get it from [NuGet](https://www.nuget.org/packages/NGitLab). You can simply install it with the Package Manager console:
 
-    PM> Install-Package NGitLab
+> PM> Install-Package NGitLab
```

Comment:
You should replace it with the dotnet sdk command

````
```
dotnet add package NGitLab
```
````